--- netmrg-0.13/etc/netmrg.xml.in.fix	2004-02-17 08:04:44.513855672 +0100
+++ netmrg-0.13/etc/netmrg.xml.in	2004-02-17 08:05:53.366388496 +0100
@@ -11,7 +11,7 @@
 		<runtimefile>@localstatedir@/log/netmrg/runtime</runtimefile>
 		<libexec>@libexecdir@/netmrg</libexec>
 		<rrds>@localstatedir@/lib/netmrg/rrd</rrds>
-		<webfileroot>@localstatedir@/www/netmrg</webfileroot>
+		<webfileroot>/home/services/httpd/netmrg</webfileroot>
 	</paths>
 	<threads>
 		<count>5</count>
